src/Native/Common/pal_config.h.in

```diff
@@ -34,6 +34,8 @@
 #cmakedefine01 HAVE_ICANON
 #cmakedefine01 HAVE_TCSANOW
 #cmakedefine01 HAVE_IN_PKTINFO
+#cmakedefine01 HAVE_IP_MREQ
+#cmakedefine01 HAVE_IP_MREQN
```

Comment:
Sure one would work. I was evaluating which one of them should be considered default (`#else` case) for OS which supports both. Since `ip_mreqn` is in majority, should I use it as default?